src/main/scala/li/cil/oc/server/component/UpgradeChunkloader.scala

```diff
@@ -142,7 +151,8 @@ class UpgradeChunkloader(val host: EnvironmentHost) extends AbstractManagedEnvir
         throw new Exception("this dimension is blacklisted")
       }
     } else {
-      ticket = Option(ForgeChunkManager.requestTicket(OpenComputers, host.world, ForgeChunkManager.Type.NORMAL))
+      // This ticket is a lie, but ChunkloaderUpgradeHandler won't crash or load it.
+      ticket = Some(new ChunkPos(0, 0))
```

Comment:
It worked in the limited testing I've done for it. The method immediately below turns it into a "real" ticket - I'm going to use vanilla's new chunk ticket system in the future, because I can have a single ticket load a 3x3 with minimal hassle.

Reply:
Hacks for the initial port beta are fine, but please add a TODO or open issue.

Reply:
Ah, I see. Cool. Yeah, ditto then :)